parcelgen.py

```diff
@@ -121,6 +128,8 @@ def gen_parcelable_line(self, typ, member):
             return self.gen_array_parcelable(typ, memberized)
         elif self.list_type(typ):
             return self.gen_list_parcelable(typ, memberized)
+        elif self.map_type(typ):
+            return self.tabify("parcel.writeMap(%s);" % (memberized))
```

Comment:
weird Android stuff, they do provide a method but they say NO, DON'T USE IT haha, oh boy, I'm switching to writing bundles even though I don't know why using writeMap() is bad (or not).
And the answer is I might know why, the design of the writeMap()/readMap() feature is really bad cause readMap() is not gonna return the map back but it asks you to provide the map as parameter already initialised, if it's null it crashes :-/ :-/ :-/